README.md

```diff
@@ -18,9 +20,9 @@ Alternatively, you can simply build the tarball (to then install it):
 
 ### Ensuring JCliff is available
 
-The collection itself only provides the integration of JCliff into Ansible. The tool needs to be installed and available on the system Ansible is running on. Refers to [JCliff](https://github.com/bserdar/jcliff) for more information on how to install the tool manually.
+The collection itself only provides the integration of JCliff into Ansible. JCliff itself needs to be installed and available on the system Ansible is running on.
 
-For commodity purpose, the Collection comes with a role 'jcliff' to have Ansible install 'jcliff'. However, currently, this role **only** supports Linux distribution using Yum (namely Fedora, RHEL and CentOS):
```

Comment:
Latest commit of Andrew includes the option to install jcliff with the help of the homebrew. So, why dont you list then MacOs within the list ?

Reply:
Because I meant to add a section on OSX, realized it was not needed and..... forgot to update the list! I'll fix that up.